src/ext_event.rs

```diff
@@ -34,6 +34,10 @@ impl ExtEventHandler {
     }
 }
 
+pub fn add_ext_trigger(trigger: Trigger) {
```

Comment:
I'm wondering if it's better to let the function to create the `Trigger` and return it?

Reply:
Probably about the same to me. But actually now that I'm thinking again I would take the trigger as an argument but call the function register_ext_trigger. 

Either works for me

Reply:
`register_ext_trigger` does look better